django/template/engine.py

```diff
@@ -68,16 +69,11 @@ def get_default():
         >>> template.render(context)
         'Hello world!'
         """
-        # Since Engine is imported in django.template and since
-        # DjangoTemplates is a wrapper around this Engine class,
-        # local imports are required to avoid import loops.
-        from django.template import engines
-        from django.template.backends.django import DjangoTemplates
-        django_engines = [engine for engine in engines.all()
-                          if isinstance(engine, DjangoTemplates)]
+        django_engines = [engine for engine in engines.templates.values()
+                          if engine['BACKEND'] == 'django.template.backends.django.DjangoTemplates']
```

Comment:
Those are not ment to be considered as default, after all you couldn't "subclass" Django template engine before the multiple template engine stuff either…